crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -176,7 +176,7 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                 transform: (context, ct) =>
                 {
                     var tableSyntax = (TypeDeclarationSyntax)context.TargetNode;
-                    var table = context.SemanticModel.GetDeclaredSymbol(tableSyntax)!;
+                    var table = context.SemanticModel.GetDeclaredSymbol(tableSyntax, ct)!;
```

Comment:
this looks like an actual runtime change, yes? Why is that?

Reply:
(I removed the `no-runtime-change` label)

Reply:
Well as it's purely codegen, technically it's still compile-time change, and one that shouldn't matter at that... but I don't have a strong opinion on that.